MobileFilter/sections/specific_web.txt

```diff
@@ -72,6 +72,7 @@ euronews.com##.amp-flying-carpet-text-border
 euronews.com##amp-fx-flying-carpet
 nu.nl##div[data-type="tealium"]
 lenta.ru###top-banner
+healthline.com##.article-body > div.css-0 > div[id^="TOC_TITLE_"] + div
```

Comment:
It must be hidden by `healthline.com##div.css-0 > div[class]:not([id])`
Please check if this rule exists in iOS filte.

Reply:
@Alex-302, i only see this rule in English and EasyList filters.